src/Jackalope/Node.php

```diff
@@ -1462,17 +1478,6 @@ public function confirmSaved()
     }
 
     /**
-     * In addition to calling parent method, tell all properties
-     */
-    public function setPath($path, $move = false)
```

Comment:
at least when we have a move operation, we must propagate that to the already instantiated properties. i think you should just do the same we have now (no need to load not yet loaded properties)